### ISSUE TITLE
Increased serum ceruloplasmin

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5953]
**Preferred term label:** Increased serum ceruloplasmin

**Synonyms**

**Definition (free text, please give PubMed ID)** An increased concentration of cerulopasmin in the blood., PMID:12055353

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK554422/

**Comment by pnrobinson** [comment]
adding as HP:0033145
